2996_reflection/reflection.md

```diff
@@ -1509,6 +1510,18 @@ std::string name1 = name_of(^S);             // Also okay.
 ```
 :::
 
+Zero-initializing an object of type `std::meta::info` gives it a null reflection value. A null reflection value is equal to any other
+null reflection value and is different from any other reflection that refers to one of the mentioned entities. For example:
```

Comment:
Let's just clarify that this is also the case for like `std::meta::info x; assert(x == x); assert(x != ^int);` kind of thing. 

And then just format to line break on sentences instead of... somewhere, which makes diffs more awkward to look at in the future. 

```suggestion
Default constructing or value-initializing an object of type `std::meta::info` gives it a null reflection value.
A null reflection value is equal to any other null reflection value and is different from any other reflection that refers to one of the mentioned entities.
For example:
```